src/server/types.ts

```diff
@@ -349,3 +349,6 @@ export interface PluginRenderFunctionResult {
    * client. */
   requiresHydration: boolean;
 }
+
+export type StartOptions = ServeInit & FreshOptions & FreshExperimantal;
+export type StartTlsOptions = FreshOptions & ServeTlsInit & FreshExperimantal;
```

Comment:
Is there a reason why `FreshOptions` comes before `ServeTlsInit` here? On the previous line `ServeInit` is before `FreshOptions`.

Reply:
no i have changed it